packages/dx-react-grid/docs/reference/table.md

```diff
@@ -33,6 +33,7 @@ columnExtensions? | Array&lt;[Table.ColumnExtension](#tablecolumnextension)&gt;
 tableComponent | ComponentType&lt;object&gt; | | A component that renders a table.
```

Comment:
"A plugin that renders Grid data as a table. It contains the Table Row and Table Cell components that can be extended by other plugins **and provides ways** to customize table rows and columns." What provides ways?